include/sigma/detail_/operation_common.hpp

```diff
@@ -5,6 +5,9 @@
 
 namespace sigma::detail_ {
 
+// Value of Pi
+constexpr double pi = 3.14159265358979323846;
```

Comment:
https://stackoverflow.com/a/1727896

Reply:
I tried a couple of versions of this, but it wasn't working out.

Reply:
But hard-coding it is really going to burn you when the C++ library updates the value... :wink:

Reply:
`M_PI` isn't part of the standard, so the value is already liable to burn you depending on library. C++20 introduces the constant pi to the standard, but `sigma` currently assumes C++17. Besides, I don't foresee changes of the value of pi within the first 20 decimal places (besides precision limitations of `double`, which is lower than 20 places).

Reply:
Sorry, I meant that comment that to be sarcastic/joking. I hope Pi doesn't change...